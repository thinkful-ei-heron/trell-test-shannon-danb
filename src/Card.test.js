import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Card from './Card'
import STORE from './STORE'

describe('rendering Card', () => {

it('renders without crashing', () => {
const div = document.createElement('div');
ReactDOM.render(<Card title='Hello' content= 'World' />, div);
ReactDOM.unmountComponentAtNode(div);
});

it('renders correctly', () => {
  const tree = renderer
    .create(<Card 
      key={STORE.allCards.a.id} 
      title={STORE.allCards.a.title} 
      content={STORE.allCards.a.content}
      />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

});