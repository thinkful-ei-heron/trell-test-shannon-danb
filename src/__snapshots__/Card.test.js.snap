// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3>
    title
  </h3>
  <p>
    content
  </p>
</div>
`;
